mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -472,6 +473,28 @@ static FuncOp createCPUConvolution(ModuleOp &module, OpBuilder &builder,
   return cpuConvFuncOp;
 }
 
+static FuncOp createVerifyKevinFuncOp(ModuleOp &module, OpBuilder &builder,
```

Comment:
the naming of the function is confusing. please revise it.

Reply:
it seems the function is not used anywhere. please remove it if that's really the case.